x/stakeibc/keeper/get_light_client_safely_test.go

```diff
@@ -15,7 +15,7 @@ func (s *KeeperTestSuite) SetupGetLightClientSafely() GetLightClientSafelyTestCa
 	s.CreateTransferChannel("GAIA")
 
 	// note this time is Jan 2020, set in the ibc test setup
-	expectedLightClientTime := int64(1577923345000000000)
+	expectedLightClientTime := int64(1577923355000000000)
```

Comment:
why was this updated?

Reply:
since the setup function changed, there was a different starting block time (I assume one extra block was produced in the startup)